runtime/queries/dart/highlights.scm

```diff
@@ -0,0 +1,259 @@
+(dotted_identifier_list) @string
+
+; Methods
+; --------------------
+(super) @function
+
+; TODO: add method/call_expression to grammar and
+; distinguish method call from variable access
+(function_expression_body (identifier) @function)
+; ((identifier)(selector (argument_part)) @function)
+
+; NOTE: This query is a bit of a work around for the fact that the dart grammar doesn't
+; specifically identify a node as a function call
+(((identifier) @function (#match? @function "^_?[a-z]"))
+  . (selector . (argument_part))) @function
```

Comment:
Please consult https://docs.helix-editor.com/themes.html#syntax-highlighting for a list of valid scopes, quite a few need changing, i.e. method->function.method etc

Reply:
I'll try to find some time for it. This was a direct copy of the linked `hightlights.scm`